virtcontainers/hypervisor.go

```diff
@@ -668,7 +668,7 @@ type hypervisor interface {
 	hypervisorConfig() HypervisorConfig
 	getThreadIDs() (vcpuThreadIDs, error)
 	cleanup() error
-	pid() int
+	pid() []int
```

Comment:
It would be nice to choose a name for pid() that clearly communicates that this method can return multiple pids.  I see there is also a getThreadIDs() method.  How about getPids() or pids() instead of pid()?